### PR TITLE
Add HTTP Sender script to RSA sign the requests

### PR DESCRIPTION
**Pull request by mwalkowski**
Hi All,

I've prepared a Python script for ZAP that adds an RSA digital signature based on a private key. I encountered such a solution in one of my projects, and I think it's worth sharing. I've also prepared a similar solution for Burp and described everything under this [link](https://mwalkowski.github.io/post/using-burp-python-scripts-to-sign-requests-with-rsa-keys/).

### CODE REVIEW

**Comment by thc202** [comment]
The changelog should be updated.

File: httpsender/rsa-signing-for-zap.py
Comment (thc202) [inline httpsender/rsa-signing-for-zap.py:1]: Ideally the name should be camel case https://github.com/zaproxy/community-scripts/blob/main/CONTRIBUTING.md#naming-scripts

Comment (mwalkowski) [reply httpsender/rsa-signing-for-zap.py:1]: Corrected

**Comment by mwalkowski** [comment]
> The changelog should be updated.

I updated changelog

**Review by psiinon — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**